rustlang/README.md

```diff
@@ -39,7 +39,8 @@ cargo install ripgrep
 
 ```bash
 cargo new hello --bin
+cd hello
```

Comment:
If you'd like to improve this:

```bash
pushd ./hello/
```

`pushd` is what modern shells actually use when you type `cd`, and it works more reliably, even cross-platform (Windows too).

Reply:
Sure, clear and explicit sounds better.

`cd` works on windows, in cmd and Powershell, however it doesn't seem to add the prior directory to whatever history pushd/popd uses, so popd does not work after a cd. Interesting.